src/preferences/colorpaletteeditor.cpp

```diff
@@ -31,7 +31,7 @@ ColorPaletteEditor::ColorPaletteEditor(QWidget* parent, bool showHotcueNumbers)
             QSizePolicy(QSizePolicy::Expanding, QSizePolicy::Preferred));
     pColorButtonLayout->addWidget(pExpander);
 
-    m_pRemoveColorButton = new QPushButton("-", this);
+    m_pRemoveColorButton = new QPushButton(QIcon::fromTheme("list-remove"), "", this);
```

Comment:
I think it might be necessary to check if the Icon is actually valid and use the text +/- as fallback.

Reply:
How? I didn't find any way to do that in the Qt documentation...........

Reply:
Oh I think I can just use QIcon::isNull after construction